package.json

```diff
@@ -13,7 +13,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "make test",
+    "test": "mocha -G --timeout 10000",
```

Comment:
We should change this to simply `mocha`, but that is outside the scope of this PR.